src/MPQEditor/MPQEditor.ts

```diff
@@ -47,6 +47,12 @@ export class MPQEditorProvider implements vscode.CustomTextEditorProvider {
         }
       ),
       // Add command registration here
+      vscode.commands.registerCommand(
+        "one.editor.mpq.showFromDefaultExplorer",
```

Comment:
May I ask why the command name is 'show*', as far as I understand, it should be 'createMPQJson'...? (THE SAME COMMENT TO https://github.com/Samsung/ONE-vscode/pull/1529)

Reply:
@dayo09 
fixed